### PR TITLE
docs: Add SOP for remote write incident management

### PR DESCRIPTION
**Pull request by philipgough**
https://issues.redhat.com/browse/RHOBS-962

### CODE REVIEW

File: docs/sop/remote_write_load_shedding_sop.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+1. Access to the Red Hat VPN.
+2. Authorization to create an MR in [app-interface](https://gitlab.cee.redhat.com/service/app-interface).
+3. Authorization to merge an MR in [app-interface](https://gitlab.cee.redhat.com/service/app-interface).
+4. Access to Grafana.
+
+## Related Alerts
+
+The following alerts can be related to remote write issues:
+
+1. [ThanosReceiveHttpRequestErrorRateHigh](https://github.com/rhobs/configuration/blob/main/docs/sop/observatorium.md#thanosreceivehttprequesterrorratehigh)
```
Comment (moadz) [inline docs/sop/remote_write_load_shedding_sop.md:29]: There is also the SLO alerts, and obs API handler alerts

File: docs/sop/remote_write_load_shedding_sop.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+## Incident Mitigation
+
+Starting with Thanos Receive, the following steps can be taken to mitigate an incident:
+
+* Ensure Thanos Receive is running and healthy.
+  * Ensure the hashring is not being OOM killed.
+    * If there are OOM events, check if traffic into the hashring has increased.
+    * If there is an unexpected increase in traffic, rate limit accordingly on observatorium-api for the tenant.
```
Comment (moadz) [inline docs/sop/remote_write_load_shedding_sop.md:38]: Secondary point, but i think we should probably document this process as well and link it here. (not an ask for right now, just future)

**Review by moadz — APPROVED** [review]
Awesome work! added some NIT's

File: docs/sop/remote_write_load_shedding_sop.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+    * If there is an expected increase in traffic, scale up the hashring with additional replicas.
+  * Ensure the hashring is not being CPU throttled.
+    * Check with the following promql expression: `rate(container_cpu_cfs_throttled_seconds_total{namespace=~"observatorium.*|rhobs", container="thanos-receive"}[5m])`
+    * If there are CPU throttling events, check if traffic into the hashring has increased.
+    * If there is an unexpected increase in traffic, rate limit accordingly on observatorium-api for the tenant.
+    * If there is an expected increase in traffic, adjust the CPU limits accordingly in the StatefulSet spec.
+
```
[the remark this answers is not in the document]
Comment (philipgough) [reply docs/sop/remote_write_load_shedding_sop.md:45]: I feel like that is getting into the obvious then and it comes to a point like where do you stop documenting things?

File: docs/sop/remote_write_load_shedding_sop.md (hunk 2 above)
Comment (philipgough) [reply docs/sop/remote_write_load_shedding_sop.md:39]: Added

**Pull request merged**